packages/web/src/server/guards/requireAdmin.ts

```
@@ -13,7 +14,19 @@ export type AdminGuardResult =
   | { ok: false; response: Response };
 
+/**
+ * Admin gate: CSRF (mutations only) → session present → role === 'admin'.
+ *
+ * The Hono mount used to apply `requireTrustedOrigin` umbrella-style across
+ * all admin routes. After the TanStack migration each admin handler calls
+ * `requireAdmin` directly, so the CSRF check is bundled here to preserve
+ * the original protection. The check internally short-circuits for
+ * GET/HEAD/OPTIONS, so admin reads are unaffected.
+ */
 export async function requireAdmin(request: Request, env: Env): Promise<AdminGuardResult> {
```

ESLint

[error] 'Env' is not defined.

(no-undef)